### PR TITLE
fix(calendar): calendar adjustments

### PR DESCRIPTION
**Pull request by Flashmaxi**

### CODE REVIEW

File: apps/web/src/components/Calendar/custom-event-month.tsx [hunk 1]
```diff
@@ -39,10 +38,11 @@ export const CustomEventMonth = ({ event }: CustomEventProps) => {
       break;
     }
   }
+  console.log('Event:', event);
```
Comment (kevinmulier) [inline apps/web/src/components/Calendar/custom-event-month.tsx]: Remove console log

**Review by kevinmulier — CHANGES_REQUESTED** [review]
Weird bug when going over some events 
[Capture vidéo du 25-11-2024 18:34:54.webm](https://github.com/user-attachments/assets/3eef3331-7216-4cbf-88ca-a83aa93606a5)

Impossible to read some events info
image

**Comment by kevinmulier** [comment]
image
bug still appears, can't read info when cards are on the end/they are too small

**Comment by Flashmaxi** [comment]
> image bug still appears, can't read info when cards are on the end/they are too small

I  think for now until we figure out different solution we need to keep it like this, I plan to keep this as ticket to try to improve react big calendar even more and if I figure out solution I will implement it. At the moment only PM/AM is not visible and I think that is not super relevant since we have calendar only on desktop at the moment and people are clicking on events and opening full detail event card.

**Pull request merged**